index.html

```diff
@@ -1,17 +1,76 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <title>Leiden Software Developers</title>
-    </head>
-    <body>
-        <h1>Leiden Software Developers</h1>
-        <p>Future home of the Leiden Software Developers group.</p>
-        <p>Join us at:</p>
-        <ul>
-            <li><a href="https://www.meetup.com/leidendevs/">Meetup</a></li>
-            <li><a href="https://leidendevs.slack.com/">Slack</a></li>
-            <li><a href="https://old.reddit.com/r/LeidenDevs">Reddit</a></li>
-        </ul>
-    </body>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Leiden Software Developers</title>
+    <style>
+        * {
+            font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, Ubuntu, roboto, noto, segoe ui, arial, sans-serif;
+            color: #111;
+        }
+
+        html {
+            font-size: 18px;
+        }
+
+        body {
+            padding: 4vw 10vw;
+        }
+
+        h1 {
+            font-weight: normal;
+        }
+
+        p {
+            line-height: 1.45;
+            max-width: 46ch;
+        }
+
+        img {
+            display: block;
+            width: 12ch;
+            margin-bottom: 3rem;
+            margin-left: -.25em;
+        }
+
+        a {
+            color: rgb(213, 12, 32);
+        }
+
+        footer {
+            margin-top: 4.35rem;
+            font-size: .875rem;
+        }
+
+        footer a {
+            color: currentColor
+        }
+
+        em {
+            color: rgb(213, 12, 32);
+            font-style: normal;
+        }
+    </style>
+</head>
+
+<body>
+    <h1>
+        <img src="/assets/logo.svg" alt="Leiden Software Developers">
+        Leiden Software Meetup
+    </h1>
+
+    <p>
+        Intended to offer a place where people who work with software, for fun or profit, can meet & socialize.
```

Comment:
An ampersand should be encoded using entities: `&amp;`

Reply:
🤗